src/cljs/harja/views/hallinta/tyokalut/talvihoitoreitit_tyokalu_nakyma.cljs

```diff
@@ -0,0 +1,135 @@
+
+  (:require-macros [cljs.core.async.macros :refer [go]]))
+
+(defn talvihoitoreittilomake [e! {:keys [talvihoitoreitti] :as app}]
+  (let [disable-tallenna? (if (nil? (:valittu-urakka talvihoitoreitti))
+                            true
+                            false)]
+    [:div.yhteydenpito
+     [:h2 "Talvihoitoreitin lähetys valitulle urakalle"]
+     [:p "Aloita valitsemalla hallintayksikkö ja sitten urakka."]
+     [lomake/lomake
+      {:ei-borderia? true
+       :tarkkaile-ulkopuolisia-muutoksia? true
+       :footer-fn (fn [talvihoitoreitti]
+                    [:div
+                     [napit/tallenna "Lähetä"
+                      #(e! (tiedot/->Laheta talvihoitoreitti))
+                      {:disabled disable-tallenna? :paksu? true}]])
+       :muokkaa! #(e! (tiedot/->Muokkaa %))}
+      [{:nimi :valittu-hallintayksikko
+        :otsikko "Valitse hallintayksikko"
+        :tyyppi :valinta
+        :valinnat @hal/vaylamuodon-hallintayksikot
+        :valinta-nayta :nimi
+        :pakollinen? true}
+       {:id (hash (:mahdolliset-urakat app))
+        :nimi :valittu-urakka
+        :otsikko "Valitse urakka"
+        :tyyppi :valinta
+        :valinnat (:mahdolliset-urakat app)
+        :valinta-nayta :nimi
+        :pakollinen? true}
+       {:nimi :valittu-jarjestelma
+        :otsikko "Järjestelma"
+        :tyyppi :string
+        :pituus-max 40
+        :pakollinen? true}
+       {:nimi :suorittaja-nimi
+        :otsikko "Suorittaja"
+        :tyyppi :string
+        :pituus-max 40
+        :pakollinen? true}
+       {:nimi :lahetysaika
+        :otsikko "Lähetysaika"
+        :tyyppi :string
+        :pituus-max 40
+        :pakollinen? true}
+       {:nimi :tyhja1
+        :tyyppi :komponentti
+        :komponentti (fn [n]
+                       [:div ""])}
+       {:nimi :ulkoinen-id
+        :otsikko "Uniikki tunniste"
+        :tyyppi :string
+        :pakollinen? true}
+       {:nimi :reittinimi
+        :otsikko "Reitin nimi"
+        :tyyppi :string
+        :pakollinen? true}
+       {:nimi :kalusto-otsikko
+        :tyyppi :komponentti
+        :komponentti (fn [n]
+                       [:div "Voit syöttää kahdet kalustot:"])}
+       {:nimi :kalusto-otsikko2
+        :tyyppi :komponentti
+        :komponentti (fn [n]
+                       [:div ""])}
+       {:nimi :kalustotyyppi
+        :otsikko "Kaluston tyyppi (1)"
+        :tyyppi :string
+        :pakollinen? true}
+       {:nimi :kalusto-lkm
+        :otsikko "Kaluston määrä (1)"
+        :tyyppi :numero
+        :pakollinen? true}
+       {:nimi :kalustotyyppi2
+        :otsikko "Kaluston tyyppi (2)"
+        :tyyppi :string
+        :pakollinen? true}
+       {:nimi :kalusto-lkm2
+        :otsikko "Kaluston määrä (2)"
+        :tyyppi :numero
+        :pakollinen? true}
+       {:nimi :tierekisteri-otsikko
+        :tyyppi :komponentti
+        :komponentti (fn [n]
+                       [:div "Voit syöttää kaksi eri tierekisteriosoitetta:"])}
```

Comment:
Saiskohan tähän helposti kartan näkyviin mistä voisi valita pisteet? Hieman manuaalista hommaa, mutta näyttää toimivan kyllä propsit siitä

Reply:
Seuraavaksi aattelin panostaa vain listaukseen, josta näkisi kaikki ympäristössä käytössäolevat tierekisteriosoitteet. Kun näitä lähinnä tarvitaan tuolla GC tms koneilla. Noissa ympäristöissä on vaikea tietää, että millainen tieosoiteverkko siellä on kannassa olemassa. 
Eli kartalta valintakaan ei välttisti auta, kun tieosoitteita on kannassa niin vähän.

Mutta. Olen siis pohtinut tuotakin.